extensions/ipynb/src/notebookSerializer.ts

```diff
@@ -70,11 +71,46 @@ export class NotebookSerializer implements vscode.NotebookSerializer {
 		return data;
 	}
 
-	public serializeNotebook(data: vscode.NotebookData, _token: vscode.CancellationToken): Uint8Array {
-		return new TextEncoder().encode(this.serializeNotebookToString(data));
+	public async serializeNotebook(data: vscode.NotebookData, _token: vscode.CancellationToken): Promise<Uint8Array> {
+		return new TextEncoder().encode(await this.serializeNotebookToString(data));
 	}
 
-	public serializeNotebookToString(data: vscode.NotebookData): string {
+	private async serializeViaWorker(workerData: notebookSerializationWorkerData): Promise<string> {
+		const workerThreads = await import('node:worker_threads');
+		const path = await import('node:path');
+		const { Worker } = workerThreads;
+
+		return await new Promise((resolve, reject) => {
+			const workerFile = path.join(__dirname, 'notebookSerializerWorker.js');
```

Comment:
this will break once the extension is bundled. We need to copy the worker file to dist. Similar to https://github.com/nojaf/vscode/blob/12abcccc8147cb30d5004fb41366b96670bdbf6c/extensions/markdown-language-features/extension.webpack.config.js#L21-L31

Reply:
Good point, I tried to add the copy plugin. But I'm not sure how I can trigger the bundling?
Is there a `yarn run` script that will bundle the extension, so I can test this?

Reply:
sorry for misleading you here, local build and product bundling are behaving differently. I pushed a commit to fix this and will give this a test.